_posts/2024-10-15-anaconda-defaults.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: Ananconda, defaults, and how to not get sued
+
+date: 2024-10-15 00:00:00
+author: "Ammar Aziz"
```

Comment:
this field isn't used anywhere, fwiw. the 'contributors' you listed pulls that out for the by-line on the post. we want a multiply valued field and `author` was always too flat for that. i should really add some frontmatter linting.

Reply:
Thanks @hexylena . Should I leave it as is or modify it somehow?

Reply:
Oh fine to leave it, doesn't hurt anything :) I'll come back and standardise all the posts someday when I add linting for the frontmatter, let's not worry about it until then.